### PR TITLE
Section reorganization

### PR DESCRIPTION
**Pull request by lpiron**
Show the new proposed baseline for Section 7 in IOP 5.0

Text is reorganized so that it is now:
1. Introduction
2. HTTPS considerations
3. Proposed section for content encryption (place holder)
4. ISOBMFF section: All about ISOBMFF and constraints that apply for all cases
5. MDP section: All about MPD and constraints that apply for all cases
6. Constraints that are both ISOBMFF and MPD related (which one has the prevalence over the other one)
7. Discussion on Client - DRM server interaction
8. Review on some specific use cases where additional constraints are added to the basic ones, including a
placeholder for low latency.

* Text has been moved from the previous version based on the general sense of the paragraphs. Additional reviews are needed for moving some remaining paragraphs not in their right places and removing redundancies. All text is only moved only, no modification except as proposed below.
* Section 7.8 (workflow overview) is proposed to be removed.

### CODE REVIEW

File: section7.bs.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ The recommendations in this document reduce the encryption parameters and use of
 
 The base technologies are introduced first followed by informative chapter on standardized elements. Additional Content Protection Constraints are then listed that are specific to conformance to DASH-264/AVC IOP.
 
+The normative standard that defines common encryption in combination with ISO BMFF is ISO/IEC 23001-7 [30].  It includes:
```
Comment (sandersaares) [inline section7.bs.md]: References to external documents should be using `[[!MPEGDASH]]` style as opposed to raw endnote links like in v4. See "references to external documents" section at https://dashif.org/DocumentAuthoring/Content.html

If some external documents are missing from SpecRef (entirely possible) they should be added there.

External documents referenced purely by name/URL should also be converted to SpecRef references (e.g. mixed content reference below).

File: section7.bs.md [hunk 2]
```diff
@@ -179,77 +134,58 @@ The following table provides pointers to relevant information in the specificati
...
+* moof/traf/sbgp						(one per sample group)
+* moof/traf/sgpd ‘seig’(sample group entry)	(one per sample group)
+* moof/pssh							(zero or one per system ID)
 
-This is a simple implementation and a possible alternative but has limitations in the flexibility:
+The main DRM components are:
 
-* The signal does not allow for early warning and time to switch the encryption keys and context.
-* The logic of the periods is decided by content creation not DRM.  Boundaries may not be suited and period may be longer than desired key interval.
+1. ‘tenc’ parameters that specify encryption parameters and default_KID (see [30] 8.2). The ‘tenc’ information is in the Initialization Segment. Any KIDs in Movie Fragment sample group description boxes override the ‘tenc’ parameter of the default_KID, as well as the ‘not encrypted’ parameter. Keys referenced by KID in sample group descriptions must be available when samples are available for decryption, and may be stored in a protection system specific header box (‘pssh’) in each movie fragment box (‘moof’). The default_KID information may also appear in the MPD (see [30] 11).
```
Comment (sandersaares) [inline section7.bs.md]: It is common to reference technical terms between backticks like this:

```
`term`
```

This gives them special "code" formatting. The characters currently used appear to be some form of Word-style single quotes, which do not apparently get any special treatment/formatting.

File: section7.bs.md [hunk 3]
```diff
@@ -179,77 +134,58 @@ The following table provides pointers to relevant information in the specificati
...
+1.	‘pssh’ license acquisition data or keys for each DRM in a format that is “Protection System Specific”. ‘pssh’ refers to the Protection System Specific Header box described in [30], 8.1.2.  ‘pssh’ boxes may be stored in Initialization or Media Segments (see [31] 8.1 and 8.2). It may also be present in a cenc:pssh element in the MPD (see [4] 5.8.4.1, [30] 11.2.1).  cenc:pssh information in the MPD allows faster parsing, earlier access, identification of duplicate license requests, and addition of DRMs without content modification.  ‘pssh’ boxes in Initialization Segments are not recommended because they trigger a license request each time an Initialization Segment is processed in a Web browser for each Representation and bitrate switch.
```
Comment (sandersaares) [inline section7.bs.md]: For all terms that are defined in the document, you should use `[=term references=]`. For example, for "initialization segment" and "media segment" and whatever other terms are used.

These would be defined as `<dfn>initialization segment</dfn>` somewhere, probably outside CPS chapter. For the time being, while only CPS chapter is present, I would just add a "placeholder definitions" subchapter to hold the definitions, to be replaced during merge with full document. Just a list of defined terms without any definitions.

**Comment by lpiron** [comment]
All good comments, thanks. I would prefer to manage all this with an editorial update separate from this change as this also needs some harmonization on the full document and looks like I will need to look at SpecRef more closely.

**Pull request merged**